libstorage/MemoryTable2.cpp

```diff
@@ -566,7 +569,7 @@ dev::storage::TableData::Ptr MemoryTable2::dump()
 
 
 #if FISCO_DEBUG
-            auto printEntries = [](Entries::Ptr entries) {
+            auto printEntries = [](const string tableName, Entries::Ptr entries) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'tableName' should be passed by const reference.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5586585)